src/js/entities-service/patient-search-results.js

```diff
@@ -0,0 +1,17 @@
+import BaseEntity from 'js/base/entity-service';
+import { Model, Collection } from './entities/patient-search-results';
+
+const Entity = BaseEntity.extend({
+  Entity: { Model, Collection },
+  radioRequests: {
+    'searchPatients:collection': 'getCollection',
+    'fetch:searchPatients': 'fetchSearchPatients',
```

Comment:
ah this should probably be `:collection`  missed that before but we also postfix those right now.